### PR TITLE
Telos Network Active repos

### PR DESCRIPTION
**Pull request by nathanduft44**
Added Telos Network active repos under telos.toml. 

Next step is to create .tomls for the sub ecosystems then add under telos sub ecosystem.

### CODE REVIEW

**Review by nathanduft44 — COMMENTED** [review]
Added in the active developer repos building on Telos chain. 

- All have their own tomls ✅
-  Proper name in Telos Sub_ecosytem used ✅

More to be added in near future. We are really interested in our developer activity.

**Comment by nathanduft44** [comment]
Has anyone taken a look at this yet?

File: data/ecosystems/t/telos.toml [hunk 1]
```diff
@@ -317,3 +307,69 @@ url = "https://github.com/telosnetwork/WETH10"
...
+url = "https://github.com/telosnetwork/safe-client-gateway"
+
+[[repo]]
+url = "https://github.com/telosnetwork/telos-evm-docker"
+
+[[repo]]
+url = "https://github.com/telosnetwork/telos.contracts"
+
+[[repo]]
+url = "https://github.com/telosnetwork/validators-nodehttps://github.com/telosnetwork/validators-node"
```
Comment (donnyquixotic) [inline data/ecosystems/t/telos.toml:375]: remove duplicate?

File: data/ecosystems/t/telos.toml [hunk 2]
```diff
@@ -297,9 +290,6 @@ url = "https://github.com/telosnetwork/tokenmanager"
 [[repo]]
 url = "https://github.com/telosnetwork/tradefortlos"
 
-[[repo]]
-url = "https://github.com/telosnetwork/ual-cleos"
-
 [[repo]]
 url = "https://github.com/telosnetwork/ual-telos-keycat"
```
Comment (donnyquixotic) [inline data/ecosystems/t/telos.toml:294]: remove this entry, no longer maintained/supported

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Comment by 0xnirmal** [comment]
@jubos @mariashen, any update on this?

File: data/ecosystems/t/telos.toml [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
+    "ApeSwap Finance",
+    "Zappy Finance",
+    "Sushi Swap",
+    "Elk Finance",
+    "PancakeSwap",
+    "Symbiosis Finance",
+    "Protofire",
+    "Actifit",
+    "API3",
+    "DIA",
```
Comment (jubos) [inline data/ecosystems/t/telos.toml:16]: Most of these aren't dedicated Telos sub ecosystem.  We have a notion of primary and secondary ecosystems for the report, but this taxonomy only reflects primary ecosystems. 

Which of these are direct Telos subecos?

File: data/ecosystems/s/symbiosis-finance.toml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+url = "https://github.com/symbiosis-finance/ssl-watch",
+
+[[repo]]
+url = "https://github.com/symbiosis-finance/assets",
+
+[[repo]]
+url = "https://github.com/symbiosis-finance/DefiLlama-Adapters",
+
+[[repo]]
+url = "https://github.com/symbiosis-finance/DefiLlama-Adapters",
```
Comment (jubos) [inline data/ecosystems/s/symbiosis-finance.toml:28]: Please fix the trailing commas

**Review by jubos — CHANGES_REQUESTED** [review]
Thanks for these ecosystems.  Looks like a great start. 

Only changes required are to remove them from the sub ecos of Telos.

**Comment by nathanduft44** [comment]
I went ahead and removed the Items from the sub-ecosystem list, temporarily until I fully understand what can be classified as a sub-eco under the Telos network. 

After taking a look at Avalanche and Solana it seems to be sub ecos must only be active on one-chain. A cross-chain DApp like Sushi Swap or ApeSwap doesn't qualify. - Is this correct please clarify. 🤔

Thanks!

**Comment by jubos** [comment]
Yes, that's correct.  

For multichain ecos, if there is a repo that is specific to one ecosystem, that can be added manually to the toml file for that ecosystem.  For the report there is what we call a "primary" vs. "secondary" sub ecosystem relationship, but that it not yet exposed in this taxonomy.  

An example would be something like Sushi where the core governance and staking primitives are on Ethereum mainnet, but it has expanded to many other ecosystems.  So that one's primary parent is Ethereum and they have secondary relationships with many other chains.

**Comment by nathanduft44** [comment]
@jubos thanks for clarifying! I have temporarily removed the sub ecos from Telos. Will add in some primary ecosystems in the coming days. Would appreciate if you could merge the latest core Telos-Network repos under Telos.toml.

**Comment by jubos** [comment]
Thanks, merged this.  Please open another PR when you have reviewed the sub ecosystems.

**Pull request merged**